src/main/java/com/takipi/oss/storage/data/status/MachineStatus.java

```diff
@@ -68,6 +70,22 @@ public long getSourceCodeSizeBytes() {
 		return sourceCodeSizeBytes;
 	}
 	
+	public void setCerebroCount(long cerebroCount) {
+        this.cerebroCount = cerebroCount;
```

Comment:
The indentation throughout this PR is incorrect, it uses spaces instead of tabs.
I'm guessing it's a configuration thing in your IDE :)